### DISCUSSION TITLE
Error message "folder loop"

### DISCUSSION THREAD

**Discussion by sircode** [floccusaddon/floccus#1794] [General]
![Screenshot_2024-12-11-14-28-59.jpg](https://github.com/user-attachments/assets/35c0912e-207e-4683-9254-1addee4c78fb)

What ever this is, should not happen in the first place or needs an explanation and solution.

**Comment by marcelklehr** [comment]
Hello @sircode 
I'm sorry this happened to you.
If you want you can open an actual issue to get someone (likely me) to look at this properly and fix this.
